core/envelope-sim/src/main/java/fr/sncf/osrd/envelope/DriverBehaviour.java

```diff
@@ -0,0 +1,30 @@
+package fr.sncf.osrd.envelope;
+
+public class DriverBehaviour {
+    public double speedTransitionOffsetIncreasing = 0;
+    public double speedTransitionOffsetDecreasing = 0;
+
+    public DriverBehaviour(double speedTransitionOffsetIncreasing, double speedTransitionOffsetDecreasing) {
+        this.speedTransitionOffsetIncreasing = speedTransitionOffsetIncreasing;
+        this.speedTransitionOffsetDecreasing = speedTransitionOffsetDecreasing;
+    }
+
+    public DriverBehaviour() {
+    }
+
+    public double getSpeedTransitionOffsetIncreasing() {
+        return speedTransitionOffsetIncreasing;
+    }
+
+    public double getSpeedTransitionOffsetDecreasing() {
+        return speedTransitionOffsetDecreasing;
+    }
+
+    public void setSpeedTransitionOffsetIncreasing(double speedTransitionOffsetIncreasing) {
+        this.speedTransitionOffsetIncreasing = speedTransitionOffsetIncreasing;
+    }
+
+    public void setSpeedTransitionOffsetDecreasing(double speedTransitionOffsetDecreasing) {
+        this.speedTransitionOffsetDecreasing = speedTransitionOffsetDecreasing;
+    }
```

Comment:
There's no need for a default constructor here.

Setters imply that the values will be changed over the lifetime of the object, which isn't the case, so they aren't needed. The fields should in fact be `final`.

I don't think the getters are really needed either, at least once the fields are set to `final`. These values aren't implementation details to be hidden away.